codebuild/common.yml

```diff
@@ -0,0 +1,11 @@
+version: 0.2
+phases:
+  install:
+    commands:
+# To be uncommented when the file is properly updated in master
+#      - python3 -c "from urllib.request import urlretrieve; urlretrieve('https://raw.githubusercontent.com/awslabs/aws-c-common/master/codebuild/builder.py', 'builder.py')"
```

Comment:
Can we just rebuild the images to have elasticurl?

Reply:
Paging mr @justinboswell